lessons/Early-Math/associative-property/langs/en-us/pages/props-make-comp-easier.adoc

```diff
@@ -0,0 +1,101 @@
+= Properties Make Computation Easier
+
+Apply the Associative Property to draw the Circle of Evaluation that will make the computation the simplest. Evaluate the expression. The first one is done for you.
+
+++++
+<style>
+  body.workbookpage table.FillVerticalSpace {
+    grid-template-rows: unset !important; grid-auto-rows: 1fr;
+  }
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+[.FillVerticalSpace, cols="^4a,^4a"]
+|===
+
+
+| @math{13 + 4 + 7 + 6}
+
+@vspace{2ex}
+
+@show{(coe '(+ (+ 13 7) (+ 4 6)))}
+
+@vspace{2ex}
+
+__30__
+
+| @math{23 + 31 + 17 + 14}
+
+@vspace{2ex}
+
+@showsoln{(coe '(+ (+ 23 17) (+ 31 14)))}
+
+@vspace{2ex}
+
+@ifsoln{@math{86}}
+
+
+| @math{125 \times 13 \times 8}
+
+@vspace{2ex}
+
+@showsoln{(coe '(* (* 125 8) 13))}
+
+@vspace{2ex}
+
+@ifsoln{@math{1300}}
+
+| @math{60 + (5 \times 74 \times 2)}
+
+@vspace{2ex}
+
+@showsoln{(coe '(+ 60 (* (* 5 2) 74)))}
+
+@vspace{2ex}
+
+@ifsoln{@math{800}}
+
+
+| @math{(15 \times 13 \times 2) + 13 \times 25 \times 4}
+
+@vspace{2ex}
+
+@showsoln{(coe '(+ (* 13 (* 15 2)) (* 13 (* 25 4))))}
+
+@vspace{2ex}
+
+@ifsoln{@math{1690}}
+
+| @math{2 + (5 \times 33 \times 2)}
```

Comment:
still not convinced we've provided the scaffolding to set them up to be successful with multiple operators